assignments/prototypes.js

```diff
@@ -32,7 +57,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(humObject) {
+  // GameObject.call(this, humObject)
+  CharacterStats.call(this, humObject )
+  this.faction = humObject.faction
+  this.weapons = humObject.weapons
+  this.language = humObject.language
+  
+}
+// Humanoid.prototype = Object.create(GameObject.prototype)
```

Comment:
Ah you found this was redundant. Can you put it in your own words why this is redundant? Add it as a comment to this thread.

Reply:
CharacterStats already carried within it the prototype inheritance from GameObject.